sympy/printing/latex.py

```diff
@@ -607,8 +607,10 @@ def _print_BasisDependent(self, expr):
         return outstr
 
     def _print_Indexed(self, expr):
-        tex = self._print(expr.base)+'_{%s}' % ','.join(
-            map(self._print, expr.indices))
+        tex_base = self._print(expr.base)
+        if re.search(r'_\{.\}$', tex_base) is not None:
```

Comment:
As I said on the issue, I would just always wrap it, rather than using a regex.

Reply:
Sir I tried to alter the code to wrap up every base in {} by removing regex  but I am getting following assertion error when I run test cases for it 
```
Traceback (most recent call last):
  File "e:\nikhil docs\sympy\sympy\printing\tests\test_latex.py", line 528, in test_latex_indexed
    or symbol_latex.split() == indexed_latex.split()[::-1]
AssertionError
```  
On investigating it further the value of  `symbol_latex='\\Psi_{0} \\overline{\\Psi_{0}}'` and `indexed_latex='\\overline{{\\Psi}_{0}} {\\Psi}_{0}'`  clearly depicts that `indexed_latex` has additional {} due to change in  `_print_Indexed(self, expr):` .In order to overcome this error either we need to change test case for this function or we should alter the following code for latex method  to include additional {} in `symbol_latex` which is as under :-
```
def _print_Symbol(self, expr):
        if expr in self._settings['symbol_names']:
            return self._settings['symbol_names'][expr]

        return self._deal_with_super_sub(expr.name) if \
            '\\' not in expr.name else expr.name
```

@asmeurer Please guide me how should I proceed further on this issue.

Regards 
Ayushman Koul